seed/challenges/04-video-challenges/jslingo.json

```diff
@@ -27,11 +27,6 @@
           "Mozilla Developer Network is a great resource that should be referenced regularly.",
           true
         ],
```

Comment:
Travis CI fails because of this comma. It is a good practice to to `npm test` before pushing commits to be sure that nothing is broken.